dbscan/structref.go

```diff
@@ -31,7 +31,9 @@ func getColumnToFieldIndexMap(structType reflect.Type) map[string][]int {
 			}
 
 			dbTag, dbTagPresent := field.Tag.Lookup(dbStructTagKey)
-
+			if dbTagPresent {
+				dbTag = strings.SplitN(dbTag, ",", 2)[0]
```

Comment:
I think we can simplify and just use `strings.Split()`
```suggestion
				dbTag = strings.Split(dbTag, ",")[0]
```

Reply:
Split means go will still continue to look for more commas when we don't care about anything past the first one.  So this is a micro optimization, if you rather use split then thats fine with me

Reply:
Yeah, I would avoid micro-optimizations in favor of simpler code. So, let's use `.Split()` here.